packages/teleport-component-generator-vue/src/vue-component.ts

```diff
@@ -1,89 +1,38 @@
-import {
-  AssemblyLine,
-  Builder,
-  Resolver,
-  Validator,
-  Parser,
-} from '@teleporthq/teleport-generator-core'
+import { createGenerator } from '@teleporthq/teleport-component-generator'
 
 import vueComponentPlugin from '@teleporthq/teleport-plugin-vue-base-component'
 import vueStylePlugin from '@teleporthq/teleport-plugin-vue-css'
-import { createPlugin as createImportStatementsPlugin } from '@teleporthq/teleport-plugin-import-statements'
+import importStatementsPlugin from '@teleporthq/teleport-plugin-import-statements'
+
+import prettierJS from '@teleporthq/teleport-postprocessor-prettier-js'
+import prettierHTML from '@teleporthq/teleport-postprocessor-prettier-html'
```

Comment:
This can help us in creating one more plugin for post-processing beautification --> https://github.com/Prettyhtml/prettyhtml/tree/master/packages/prettyhtml-formatter 🎉